### PR TITLE
Vast storm

### PR DESCRIPTION
**Pull request by otaiwo70**

### CODE REVIEW

File: backend/game/cards/discard_cards/multi_manipulation_cards/test_vast_storm.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+def test_vast_storm_adds_2_victory_points(player):
+    VastStorm().immediate_effect(player, None)
+    assert player.victory_points == 2
+
+
+def test_vast_storm_subtracts_3_health(player):
+    player.current_health = 8
+    VastStorm().immediate_effect(player, None)
+    assert player.current_health == 7
```
Comment (gpfaia) [inline backend/game/cards/discard_cards/multi_manipulation_cards/test_vast_storm.py]: This math doesn't add up to the test name

Comment (otaiwo70) [reply backend/game/cards/discard_cards/multi_manipulation_cards/test_vast_storm.py]: vast storm removes 1 health point, set initial to 8 then trigger method which removes 1 health point outcome is 7

**Comment by borisv13** [comment]
You were supposed to go through all of the test cases and change this to
remove the initial set, and instead just assert on DEFAULT_HEALTH-1.

File: backend/game/cards/discard_cards/multi_manipulation_cards/test_vast_storm.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from game.cards.discard_cards.multi_manipulation_cards.vast_storm import VastStorm
+from game.values import constants
+
+
+def test_vast_storm_adds_2_victory_points(player):
+    VastStorm().immediate_effect(player, None)
+    assert player.victory_points == 2
+
+
+def test_vast_storm_subtracts_3_health(player):
```
Comment (borisv13) [inline backend/game/cards/discard_cards/multi_manipulation_cards/test_vast_storm.py]: subtracts 3 or subtracts 1?

Comment (otaiwo70) [reply backend/game/cards/discard_cards/multi_manipulation_cards/test_vast_storm.py]: typo to test name rectified

**Pull request merged**